cmd/tailscale/cli/configure.go

```diff
@@ -32,5 +33,6 @@ func configureSubcommands() (out []*ffcli.Command) {
 		out = append(out, synologyConfigureCmd)
 		out = append(out, synologyConfigureCertCmd)
 	}
+	out = slicesx.AppendNonzero(out, []*ffcli.Command{maybeSysExtCmd, maybeVPNConfigCmd})
```

Comment:
if you used `nonNilCmds` on line 28 you could simplify this and maybe drop this whole line 31 function